speaker/main.go

```diff
@@ -61,13 +62,15 @@ type service interface {
 func main() {
 	prometheus.MustRegister(announcing)
 
+	const mlSecretKeyName = "secretkey"
```

Comment:
Having this as a constant while it's used only in one place is not adding value. On the other hand, I think it makes sense to have it as a global constant in k8s.go and use it also there (where we create the secret).

Reply:
done! thanks